flatdata-go/backward-compatibility-tests/Gopkg.lock

```diff
@@ -22,6 +22,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "a9dd4821c4522b2069722c1be94b191e7aff1736aeb5c12ab0070f87cdbc5af2"
+  inputs-digest = "0a81cdd8b28938307cd5d3941bfdb99c2b2a04aac9c0cb95558335aaab9ea4bd"
```

Comment:
@artemnikitin  Should this be updated?

Reply:
No need, it's up to date in master
https://github.com/heremaps/flatdata/blob/master/flatdata-go/backward-compatibility-tests/Gopkg.lock#L25